src/components/Picker/Picker.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { Text } from '../Text';
+import { PICKER_DATA_HOOKS, PICKER_DATA_KEYS } from './dataHooks';
+import { ReactComponent as ChevronLeft } from '../../assets/icons/ChevronLeft.svg';
+import { ReactComponent as ChevronRight } from '../../assets/icons/ChevronRight.svg';
+import { IconButton } from '../IconButton';
+import styles from './Picker.st.css';
+
+export interface PickerProps {
+  value: any;
+  previousClickHandler(): void;
+  nextClickHandler(): void;
+  arrowsSize?: string;
+}
+
+interface DefaultProps {
+  arrowsSize: string;
+  'data-hook': string;
+}
+
+/** Picker */
+export class Picker extends React.Component<PickerProps> {
```

Comment:
I think `Picker` is a bit too general.
Let's call it `CalendarPicker`

Reply:
The picker should be general. i don't see the behavior and style change for any use case.